### PR TITLE
TS-4162: Add proxy.config.http2.active_timeout_in

### PR DESCRIPTION
**Pull request by masaori335**
- Fix [TS-4162](https://issues.apache.org/jira/browse/TS-4162)

### CODE REVIEW

File: mgmt/RecordsConfig.cc [hunk 1]
```diff
@@ -1995,6 +1995,8 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.http2.no_activity_timeout_in", RECD_INT, "115", RECU_DYNAMIC, RR_NULL, RECC_STR, "^[0-9]+$", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.http2.active_timeout_in", RECD_INT, "900", RECU_DYNAMIC, RR_NULL, RECC_STR, "^[0-9]+$", RECA_NULL}
```
Comment (masaori335) [inline mgmt/RecordsConfig.cc]: I'm not sure how log is suitable for default value. I chose `900` to make it same as `proxy.config.http.transaction_active_timeout_in`.
Any suggestions?

**Comment by zwoop** [comment]
I'm mostly ok with this, except for the default value. You should for now set it to a value (likely 0?) that is compatible with existing 6.x code. Then file a separate Jira for 7.0.0, to change the default value (and mark it as incompatible, so we know to put it into the release notes for 7.0.0).

**Comment by masaori335** [comment]
Current value is 0, because active timeout of VConnection in Http2ClientSession is not set.

I changed default value of `proxy.config.http2.active_timeout_in` to 0 for compatibility, and filed a new ticket as [TS-4167](https://issues.apache.org/jira/browse/TS-4167).

**Comment by bryancall** [comment]
Looks good.

**Pull request closed without merging**